sdk/textanalytics/ai-text-analytics/review/ai-text-analytics.api.md

```diff
@@ -546,6 +604,11 @@ export interface TargetSentiment {
     text: string;
 }
 
+// @public
+export interface TextAnalyticsAction {
+    actionName?: string;
```

Comment:
Curious, are you waiting for the service deployment in order to add this property to tests and samples?